multiapi-engine/src/main/java/com/sngular/api/generator/plugin/commonApi/Constants.java

```diff
@@ -0,0 +1,77 @@
+package com.sngular.api.generator.plugin.commonApi;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public final class Constants {
+
+  public static final String INTEGER = "integer";
```

Comment:
A doubt: won't it be better to group the constant values depending on their objective?
I'm thinking in an enum group with all the types (integer, double, float, ... as we get in `UNION`) instead of getting a big group with a constant mix.

Anyway, I think that, at least in this file, there is a style issue in line 15 (dowuble space between `static` and `final`)